test/fail_compilation/imports/diag12598a.d

```diff
@@ -1,3 +0,0 @@
-module imports.diag12598a;
-
-struct lines { }
```

Comment:
It doesn't test for anything novel anymore and serves no purpose. Hence, deleting it.